src/usr/lib/networkd-dispatcher/routable.d/google_hostname.sh

```diff
@@ -12,7 +12,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License
-
+if [[ $(jq .State <<< $json) == *"unmanaged"* ]]; then
+	exit 0
+fi
 new_host_name=$(curl -H 'Metadata-Flavor: Google' http://169.254.169.254/computeMetadata/v1/instance/hostname)
 new_ip_address=$ADDR new_host_name=$new_host_name google_set_hostname
```

Comment:
is it valid to use IP from MDS `instance/network-interfaces/0/ip` same like what we're doing [here](https://github.com/GoogleCloudPlatform/guest-configs/blob/2b9308d12e0da9837c474f1839c1ec8597712f67/src/etc/sysconfig/network/scripts/google_up.sh#L17)  instead of `ADDR`. If so, we wouldn't need this extra logic for skipping unmanaged nic right?

Reply:
Yes but it impacts boot time, I will ping you with details.

Reply:
Done.